source/schema-configuration.txt

```diff
@@ -103,6 +133,40 @@ to update the schema.
 
 .. include:: /includes/fact-resample-schema-data.rst
 
+User Permissions
+~~~~~~~~~~~~~~~~
+
+To run |bi-short| in persisted schema mode, a MongoDB user needs
+permission to create a schema from any database in the deployment and
+read and write entries into the schema database. Give this user the
+:authrole:`readAnyDatabase` role and the :authrole:`readWrite` role on
+the database that stores the schema (set using :option:`--sampleMode`)
+
+.. example::
+
+   Use the MongoDB shell to add a user to the ``admin`` database with the proper
+   permissions to read any database and read and write the schema
+   database.
+
+   .. code-block:: javascript
+      :class: copyable
+ 
+      db.getSiblingDB("admin").createUser(
+        {
+          user: "<username>",
+          pwd: "<password>",
+          roles: [
+            "readAnyDatabase",
+            { authrole: "readWrite", db: "<schemaDb>" }
+          ]
+        }
+      )
+
```

Comment:
Hey Tony -- it just occurred to me there's a syntax error here. You want "role", not "authrole", on line 160. As is it will throw an error.